frontend/test/metabase/scenarios/visualizations/scatter.cy.spec.js

```diff
@@ -0,0 +1,76 @@
+import { restore, visitQuestionAdhoc } from "__support__/cypress";
+import { SAMPLE_DATASET } from "__support__/cypress_sample_dataset";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATASET;
+
+const testQuery = {
+  database: 1,
+  query: {
+    "source-table": ORDERS_ID,
+    aggregation: [["count"], ["distinct", ["field-id", ORDERS.PRODUCT_ID]]],
+    breakout: [["datetime-field", ["field-id", ORDERS.CREATED_AT], "month"]],
+  },
+  type: "query",
+};
+
+describe("scenarios > visualizations > scatter", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
```

Comment:
Just a word of caution - normal user is not allowed to do some (more sensitive) requests. For now, this works but you may find yourself in a situation where you need more complicated preparation for some test. In that case, you'd need to use `cy.signInAsAdmin()` - which is what we use most of the time.

For some specific areas (permissions, sandboxing, etc.) it is extremely important to run the same test against many different user roles. But even then admin creates the setup for the test, and then we can sign is as a desired user role for the assertion phase.

No need to change anything here, btw. I thought this is just a good place to tell you about this :)

Reply:
Good to know, thanks for sharing